src/styles.css

```diff
@@ -0,0 +1,133 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul,
+p {
+  margin: 0;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+body {
+  font-weight: 400;
+  font-style: normal;
+  color: #060b35;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding-left: 0;
+
+  list-style: none;
+  border-radius: var(--border-radius);
+  border: 1px solid #f3f3f3;
+
+  --card-margin: 0 16px 16px;
+  --border-radius: 5px;
+}
+
+.card__img {
+  margin: 32px 19px 40px;
+}
+
+.card__info,
+.card__rating,
+.card__price {
+  margin: var(--card-margin);
+}
+
+.card__info p:first-child {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  margin: 0 0 4px;
+}
+
+.card__info p:last-child {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
```

Comment:
Add classes to this paragraphs as Jamal advised and change selectors in lines 47 and 54 to only classes. For example `card__title`